src/test_l1_dsHdmiIn.c

```diff
@@ -417,6 +427,7 @@ void test_l1_dsHdmiIn_positive_dsHdmiInGetStatus(void) {
  * 
  */
 void test_l1_dsHdmiIn_negative_dsHdmiInGetStatus(void) {
+
     gTestID = 8;
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID); 
     dsHdmiInStatus_t status;
```

Comment:
Initialize the variable dsHdmiInStatus_t status to its default value as follows: dsHdmiInStatus_t status = dsERR_NONE;. Check throughout the file to ensure all variables are initialized to their default values, including setting uint8_t numInputs = 0 and initializing any struct elements to 0, such as dsVideoPortResolution_t resolutions = {0};.

Reply:
Done